### ISSUE TITLE
firedrake-zenodo fail?

### ISSUE THREAD

**Issue by miklos1** [firedrakeproject/firedrake#720]
```
$ firedrake-zenodo --release -m 'Test Zenodo bugfix'
Failed to retrieve git hash for firedrake
Traceback (most recent call last):
  File "/home/mh1714/firedrake/bin/firedrake-zenodo", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/mh1714/firedrake/src/firedrake/scripts/firedrake-zenodo", line 164, in <module>
    shas = collect_repo_shas()
  File "/home/mh1714/firedrake/src/firedrake/scripts/firedrake-zenodo", line 125, in collect_repo_shas
    os.chdir(src + "/" + component)
OSError: [Errno 2] No such file or directory: '/home/mh1714/firedrake/src/firedrake/scripts/../src/firedrake'
```

**Comment by miklos1** [comment]
firedrake/bin/firedrake-zenodo:
```
#!/home/mh1714/firedrake/bin/python
# EASY-INSTALL-DEV-SCRIPT: 'firedrake===0.13.0-59-gaac27a2','firedrake-zenodo'
__requires__ = 'firedrake===0.13.0-59-gaac27a2'
__import__('pkg_resources').require('firedrake===0.13.0-59-gaac27a2')
__file__ = '/home/mh1714/firedrake/src/firedrake/scripts/firedrake-zenodo'
exec(compile(open(__file__).read(), __file__, 'exec'))
```

This overwrites `__file__` and thus works in an identical manner to calling firedrake-zenodo in the source tree directly.

**Comment by miklos1** [comment]
Fixed by #721.
